pkg/platform/mock/platform.go

```diff
@@ -0,0 +1,220 @@
+package mock
+
+import (
+	"github.com/nuclio/nuclio/pkg/containerimagebuilderpusher"
+	"github.com/nuclio/nuclio/pkg/platform"
+	"github.com/nuclio/nuclio/pkg/platformconfig"
+	"github.com/nuclio/nuclio/pkg/processor/build/runtime"
+
+	"github.com/stretchr/testify/mock"
+)
+
+//
+// Platform mock
+//
+
+// Platform defines the interface that any underlying function platform must provide for nuclio
+// to run over it
+type Platform struct {
+	mock.Mock
+}
+
+//
+// Function
+//
+
+// Build will locally build a processor image and return its name (or the error)
+func (p *Platform) CreateFunctionBuild(createFunctionBuildOptions *platform.CreateFunctionBuildOptions) (*platform.CreateFunctionBuildResult, error) {
```

Comment:
```suggestion
func (mp *Platform) CreateFunctionBuild(createFunctionBuildOptions *platform.CreateFunctionBuildOptions) (*platform.CreateFunctionBuildResult, error) {
```
To keep it clear this is a mock platform throughout the code (using mock interface)